### PR TITLE
Reformat clear with checklist and run usedata

### PR DESCRIPTION
**Pull request by ErinCain**

### CODE REVIEW

**Comment by ErinCain** [comment]
Cleaned up code

**Comment by ErinCain** [comment]
cleaned up code

File: data-raw/watershed/clear/clear_creek.rmd [hunk 1]
```diff
@@ -1,36 +1,28 @@
...
...dlife Service (USFWS) instream flow evaluations conducted between 2007 and 2013 for the Central Valley Project Improvement Act (CVPIA). These evaluations developed relationships between flow and suitable spawning and rearing habitat area for Clear Creek for "upper" (Whiskeytown Dam to Clear Creek Road) and "lower" (Clear Creek Road to the Sacramento River) Clear Creek. The original data can be found on pages 113 - 117 of FWS 2007 (Upper Clear Creek spawning), pages 144 - 157 of FWS Rearing 2011 (Upper Clear Creek rearing, pages 107 - 109 of FWS Spawning 2011 (Lower Clear Creek spawning), and pages 150 - 160 of FWS 2013 (Lower Clear Creek rearing). [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service combined the habitat data from these four studies and provided them for the entire Clear Creek watershed in a spreadsheet (see ‘~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx’). Mark Gard instructed us to use A3:G26, A30:G53, and A58:G81 of the 'Clear' tab within his 'IFIMWUA.xlsx'. These cells represent the WUA in sqft for the lower alluvial (8.81 mi), canyon (7.33 mi), and upper alluvial (2.27 mi) segments. Fall run are only in the lower alluvial segment.
```
Comment (thesadie) [inline data-raw/watershed/clear/clear_creek.rmd]: Will you update the text to include "Fall Run spawning is an estimate provided by Clear Creek watershed experts after receiving feedback that the FWS estimates were too low." and remove any reference that doesn't make sense given this

File: data-raw/watershed/clear/clear_creek.rmd [hunk 2]
```diff
@@ -99,170 +83,90 @@ sr_spawn <- bind_rows(clear_raw_la, clear_raw_c, clear_raw_ua) %>%
...
+  ggplot(aes(flow_cfs, acres, color = species)) +
   geom_line() +
-  labs(x = 'Flow (cfs)', y = 'WUA (sqft/1000ft)') +
-  scale_y_continuous(labels = comma) +
-  theme_minimal()+
-  scale_color_manual(values = c('#d95f02','#7570b3'))
+  theme_minimal() +
+  facet_wrap(~lifestage, ncol = 1, scales = 'free_y') +
+  scale_color_brewer(palette = 'Dark2') +
+  labs(x = 'Flow (cfs)', y = 'Suitable Acres') +
```
Comment (thesadie) [inline data-raw/watershed/clear/clear_creek.rmd]: the y axis label should be  'WUA (sqft/1000ft)', right?

Comment (ErinCain) [reply data-raw/watershed/clear/clear_creek.rmd]: fixed

File: data-raw/watershed/clear/clear_creek.rmd (hunk 1 above)
Comment (ErinCain) [reply data-raw/watershed/clear/clear_creek.rmd]: fixed

**Pull request merged**